test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+  function() external [] s0;
+  function() external [] s1;
+  function copyDynamicallySizedStorageArray() public {
+    s0 = s1;
+  }
+
+}
+contract copyDynamicArrays {
+  C public testContract;
+
+  constructor() {
+    testContract = new C();
+  }
+
+  function tryCopyingDynamicallySizedStorageArray() public returns (uint ret) {
+    try testContract.copyDynamicallySizedStorageArray() {
+      return 1;
+    } catch {
+      return 0;
+    }
+  }
+}
+// ====
+// compileViaYul: also
+// ----
+// tryCopyingDynamicallySizedStorageArray() -> 1
```

Comment:
Actually, this whole contract seems superfluous. An instance of the contract gets created automatically and you can just call the function on it in expectations:
```suggestion
// ====
// compileViaYul: also
// ----
// copyDynamicallySizedStorageArray() ->
```

Reply:
Overall most of my comments about this test apply to the other one too.